lib/gollum-lib/redirects.rb

```diff
@@ -0,0 +1,28 @@
+require 'yaml'
+
+REDIRECTS_FILE = '.redirects.gollum'
+
+module Gollum
+  
+  class Redirects
+        
+    def self.load(wiki)
+      file = wiki.file(REDIRECTS_FILE)
+      redirects = {}
+      if file
+        begin
+          redirects = YAML.load(file.raw_data)
+        rescue YAML::Error
+          # TODO handle error
```

Comment:
Method now returns `nil` when an exception is rescued. Will that go down well in gollum?

Reply:
It shouldn't, since `redirects` is returned and `redirects` is initialized as `{}`.